CHANGELOG.md

```diff
@@ -1,35 +1,136 @@
 # NEXT_VERSION
 
-* Merge "Places/Towns" and "Places/Settlements" layers into one "Populated
-  places" layer. Scale point markers for this layer by population.
-* Add new "Internet-required data/Geological Map (1:500 000)" layer from GEUS.
-* Add new "Geology/Mineral occurrences" layer from GEUS.
-* Update `layer_list.csv` to include new column indicating if each layer is
+- Resolve `Font "Helvetica" font not available on system` warnings
+- Update `layer_list.csv` to include new column indicating if each layer is
   stored on disk. Internet-required layers take the value `False`.
-* Add new "Glaciology/Ice sheet velocity" layers:
-  * "GrIMP/Annual ice sheet velocity vectors 2021"
-  * "GrIMP/Annual ice sheet velocity magnitude 2021 (200m)"
-  * "GrIMP/Annual ice sheet velocity x component 2021 (200m)"
-  * "GrIMP/Annual ice sheet velocity y component 2021 (200m)"
-* Improve symbology for "Regional climate models/RACMO model output/Annual mean
+- Establish a more explicit/readable pattern for defining the order of QGreenland's
+  layers and groups in the Layers Panel
```

Comment:
We removed some other elements from the CHANGELOG that were not 'user' facing. Should we include this? Same w/ the point below about publishing Docker images.

Reply:
Good point. I agree with removing this one, but think the point about docker images for reproducibility is arguably interesting to users.

Reply:
cb58db57